### ISSUE TITLE
branch: gameScreen

### ISSUE THREAD

**Issue by chrisklus** [phetsims/number-play#61]
@Luisav1 and i are going to develop the game screen for Number Play on this branch. It will be deleted once the game is complete and merged in.

**Comment by chrisklus** [comment]
The game screen was merged into master. If we need to publish a version of Number Play without the Game screen in the future, we can just omit it like we did with the Lab screen. Closing
